tests/cypress/template/vocab-home.cy.js

```diff
@@ -1,42 +1,134 @@
 describe('Vocabulary home page', () => {
   it('contains vocabulary title', () => {
-    cy.visit('/test/en') // go to the "Test ontology" home page
+    cy.visit('/yso/en') // Go to the "YSO - General Finnish ontology (archaeology)" home page
```

Comment:
The URL should be `/yso/en/` with the final slash included. (This was also wrong in the previous version)
The URL without the final slash will work, but there is an extra redirect because it's not the "canonical" URL of the vocabulary home page.